tools/migration/db/pgsql_migrator.py

```diff
@@ -0,0 +1,174 @@
+    write_sequence(pgsql_dump, "user_group", "id")
+    write_sequence(pgsql_dump, "access_log", "log_id")
+    write_sequence(pgsql_dump, "repository", "repository_id")
+    write_sequence(pgsql_dump, "replication_policy", "id")
+    write_sequence(pgsql_dump, "replication_target", "id")
+    write_sequence(pgsql_dump, "replication_immediate_trigger", "id")
+    write_sequence(pgsql_dump, "img_scan_job", "id")
+    write_sequence(pgsql_dump, "img_scan_overview", "id")
+    write_sequence(pgsql_dump, "clair_vuln_timestamp", "id")
+    write_sequence(pgsql_dump, "properties", "id")
+    write_sequence(pgsql_dump, "harbor_label", "id")
+    write_sequence(pgsql_dump, "harbor_resource_label", "id")
+
+def do_parse_notary_server_db(mysql_dump_file, pgsql_dump_file):
+    mysql_dump = open(mysql_dump_file)
+    pgsql_dump = open(pgsql_dump_file, "w")
+    insert_lines = []
+
+    for i, line in enumerate(mysql_dump):
+        
+        line = line.decode("utf8").strip()
+        if drop_line(line):
+            continue
+
+        # catch insert
+        if line.startswith("INSERT INTO"):
+            if line.find("\\") != -1:
+                line = line.replace('\\', '')
+                insert_lines.append(line)
+                continue
+            elif line.startswith('INSERT INTO "change_category"') | line.startswith('INSERT INTO "changefeed"') | line.startswith('INSERT INTO "schema_migrations"'):
+                continue
+            elif line.find("0000-00-00 00:00:00") != -1:
+                line = line.replace("0000-00-00 00:00:00", "0001-01-01 00:00:00")
+                insert_lines.append(line)
+                continue
+            else:    
+                insert_lines.append(line)
+    
+    write_database(pgsql_dump, "notaryserver")
+    write_insert(pgsql_dump, insert_lines)
+    write_sequence(pgsql_dump, "tuf_files", "id")
+
+def do_parse_notary_signer_db(mysql_dump_file, pgsql_dump_file):
+    mysql_dump = open(mysql_dump_file)
+    pgsql_dump = open(pgsql_dump_file, "w")
+    insert_lines = []
+
+    for i, line in enumerate(mysql_dump):
+        
+        line = line.decode("utf8").strip()
+        if drop_line(line):
+            continue
+
+        # catch insert
+        if line.startswith("INSERT INTO"):
+            if line.find("\\") != -1:
+                line = line.replace('\\', '')
+                insert_lines.append(line)
+                continue
+            elif line.startswith('INSERT INTO "schema_migrations"'):
+                continue
+            elif line.find("0000-00-00 00:00:00") != -1:
+                line = line.replace("0000-00-00 00:00:00", "0001-01-01 00:00:00")
+                insert_lines.append(line)
+                continue
+            else:    
+                insert_lines.append(line)
+    
+    write_database(pgsql_dump, "notarysigner")
+    write_insert(pgsql_dump, insert_lines)
+    write_sequence(pgsql_dump, "private_keys", "id")
+
+def drop_line(line):
```

Comment:
it's not needed if all we wanna catch is `INSERT INTO` this method is not needed.